<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CURRENT MASS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL MASS</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTimer">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>51</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLed" name="qLed_2">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed_7">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed_4">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed_6">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed_5">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed_3">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLed" name="qLed">
       <property name="toolTip">
        <string>Binary Led</string>
       </property>
       <property name="whatsThis">
        <string>Led widget</string>
       </property>
       <property name="value">
        <bool>false</bool>
       </property>
       <property name="onColor">
        <enum>QLed::Red</enum>
       </property>
       <property name="offColor">
        <enum>QLed::Grey</enum>
       </property>
       <property name="shape">
        <enum>QLed::Circle</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>111</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="buttonTeleoperation">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TELEOPERATION</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelWebCam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>421</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>CAMERA</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="text">
     <string>MAP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQuit">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>560</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>300</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHalt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>HALT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCalibrate">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>121</width>
      <height>97</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditKeepU">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditSteerW"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditKeepI"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEditSteerNr">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QSlider" name="horizontalSliderArmPosition">
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonSteer">
       <property name="text">
        <string>DO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="buttonKeep">
       <property name="text">
        <string>DO</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="buttonArmPosition">
       <property name="text">
        <string>DO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>115</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Steer Engine i, w</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Keep max U, I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Arm position</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLed</class>
   <extends>QWidget</extends>
   <header>qled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
